src/stores/ClientPresenceStore.js

```diff
@@ -22,11 +22,19 @@ class ClientPresenceStore extends PresenceStore {
   async setClientPresence(presence) {
     const packet = await this._parse(presence);
```

Comment:
One quick thing.. I don't know if Discord ignores that `shard` field, but I'd recommend running a `delete packet.shard` before, just to be sure

Reply:
Ahh packet.shard is the ID of the shard for Presence, no need to remove it